wgsl/index.bs

```diff
@@ -11985,6 +11985,38 @@ Note: The result is not mathematically meaningful when `abs(e)` &ge; 1.
     <td>Returns the dot product of `e1` and `e2`.
 </table>
 
+### `dot4U8Packed` ### {#dot4U8Packed-builtin}
+<table class='data builtin'>
+  <tr algorithm="dot4U8Packed">
+    <td style="width:10%">Overload
+    <td class="nowrap"><xmp highlight=rust>
+@const fn dot4U8Packed(e1: u32,
+                       e2: u32) -> u32
+</xmp>
+  <tr>
+    <td>Description
+    <td> `e1` and `e2` are interpreted as vectors with four 8-bit unsigned integer components.
+    Vector components follow byte significance order with the lowest-numbered component
```

Comment:
The component ordering is not important, as long as the same ordering is used to decompose both arguments.